examples/MQ/pixelDetector/src/PixelGeo.h

```diff
@@ -16,8 +16,9 @@
 class PixelGeo : public FairGeoSet
 {
   protected:
-    char modName[20]{"Pixel"};   // name of module
-    char eleName[20]{"Pixel"};   // substring for elements in module
+    static constexpr int maxbuf{20};   //!
```

Comment:
Is `maxbuf` actually our naming convention for this sort of constants?

And TBH: I don't know, whether this constant actually improves things?